examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py

```diff
@@ -123,48 +129,48 @@ def define_and_get_arguments(args=sys.argv[1:]):
 def __evaluate_models_on_test_data(test_loader, results):
     data, target = test_loader.__iter__().next()
     logger.info("Testing individual models on test data")
-    hist, bin_edges = np.histogram(target, bins=10, range=(0, 10))
+    hist, _ = np.histogram(target, bins=10, range=(0, 10))
     logger.info("Target hist: %s", hist)
-    for worker_id, worker_model, worker_loss in results:
+    for worker_id, worker_model, _ in results:
         pred_test = worker_model(data)
         loss = loss_fn(output=pred_test, target=target)
         logger.info(
             "Worker %s: Loss: %s, accuracy = %s", worker_id, loss, utils.accuracy(pred_test, target)
         )
         pred = pred_test.argmax(dim=1)
-        hist, bin_edges = np.histogram(pred, bins=10, range=(0, 10))
+        hist, _ = np.histogram(pred, bins=10, range=(0, 10))
         logger.info("Worker %s: Predicted hist: %s", worker_id, hist)
 
 
 def evaluate_models_on_test_data(test_loader, results):
     np.set_printoptions(formatter={"float": "{: .0f}".format})
-    for worker_id, worker_model, worker_loss in results:
+    for worker_id, worker_model, _ in results:
         evaluate_model(worker_id, worker_model, "cpu", test_loader)
 
 
 def evaluate_model(worker_id, model, device, test_loader, print_target_hist=False):
     model.eval()
-    test_loss = 0
+    test_loss = 0.0
     correct = 0
     hist_target = np.zeros(10)
     hist_pred = np.zeros(10)
 
     with torch.no_grad():
         for data, target in test_loader:
             data, target = data.to(device), target.to(device)
-            hist, bin_edges = np.histogram(target, bins=10, range=(0, 10))
+            hist, _ = np.histogram(target, bins=10, range=(0, 10))
             hist_target += hist
             output = model(data)
             test_loss += loss_fn(output, target).item()  # sum up batch loss
             pred = output.argmax(dim=1, keepdim=True)  # get the index of the max log-probability
-            hist, bin_edges = np.histogram(pred, bins=10, range=(0, 10))
+            hist, _ = np.histogram(pred, bins=10, range=(0, 10))
             hist_pred += hist
             correct += pred.eq(target.view_as(pred)).sum().item()
 
     test_loss /= len(test_loader.dataset)
 
     if print_target_hist:
-        logger.info("Target hist    : %s", hist_target)
+        logger.info("Target hist: %s", hist_target)
```

Comment:
The spaces are voluntary to align the output of target hist and prediction hist for easier comparison.

Reply:
Got it, can you add a comment for this?